### PR TITLE
[CDS-101821] CD release notes 1.60

### PR DESCRIPTION
**Pull request by vishal-av**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by krishi0408 — DISMISSED** [review]

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -47,6 +47,24 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+#### New features and enhancements
+
+- The Harness GitOps agent uses the **Horizontal Pod Autoscaler** for CPU and memory management, with a minimum of 1 replica and a maximum of 5 replicas in High Availability (HA) mode. (CDS-100830)
+
+#### Fixed issues
+
+- Previously, some users were unable to use GitEx bidirectional sync with Harness repositories due to the presence of special characters in the repository. This issue is resolved. Users can now create webhooks even if their repository contain special characters.(PIPE-22238, ZD-70182)
+
+- The Git experience repository search was not yielding the expected results. This issue has been resolved. The search functionality  works only with repository names, not with full paths or subdirectories. (PIPE-22173, ZD-70809)
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:60]: @vishal-av can a screenshot be added here?

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -47,6 +47,24 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- Previously, some users were unable to read service and job names from the exported manifest due to data masking. This issue is resolved. (CDS-101472, ZD-70697)
```
Comment (shylajasun) [inline release-notes/continuous-delivery.md:62]: @vishal-av which service type is this?

File: release-notes/continuous-delivery.md (hunk 1 above)
Comment (vishal-av) [reply release-notes/continuous-delivery.md:60]: yes, i will add a screenshot

**Review by SushrutHarness — DISMISSED** [review]
LGT<

**Review by wei-harness — DISMISSED** [review]

**Review by krishi0408 — DISMISSED** [review]

**Review by SudarshanPurohit — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -47,6 +47,38 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
 
 ## October
 
+### Version 1.60.5
+
+#### New features and enhancements
+
+- The Harness GitOps agent uses the **Horizontal Pod Autoscaler** for CPU and memory management, with a minimum of 1 replica and a maximum of 5 replicas in High Availability (HA) mode. (CDS-100830)
```
Comment (acasian-harness) [inline release-notes/continuous-delivery.md:54]: can we have a link to Harness docs for the feature? this applies to all features delivered @SudarshanPurohit

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -47,6 +47,58 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+
+- The ECR token was revealed through artifact expressions in the shell script step. This issue is resolved. (CDS-101258, ZD-70269)
+
+#### Gitops Version 1.18.0
+
+#### New features and enhancements
+
+- The Harness GitOps agent uses the **Horizontal Pod Autoscaler** for CPU and memory management, with a minimum of 1 replica and a maximum of 5 replicas in High Availability (HA) mode. For more information, go to [GitOps documentation](/docs/continuous-delivery/gitops/connect-and-manage/install-a-harness-git-ops-agent/#high-availability-ha). (CDS-100830)
+
+- Harness GitOps now supports Multi-Source applications with ArgoCD. This feature is available for Gitops agent version 1.18.0 onwards. Currently, this feature is behind the feature flag  `GITOPS_MULTI_SOURCE_ENABLED`. Please contact [Harness support](mailto:support@harness.io) to enable this feature. (CDS-85518)
```
Comment (acasian-harness) [inline release-notes/continuous-delivery.md:80]: this requires gitops-agent 0.79 version, we need to mention this. Also @SudarshanPurohit do we have any ref to Harness docs explaining the feature?

Comment (vishal-av) [reply release-notes/continuous-delivery.md:80]: @acasian-harness, we don't have the reference docs yet. I will add the docs and update the RN with a ref-link.

**Review by SudarshanPurohit — APPROVED** [review]

**Review by krishi0408 — APPROVED** [review]

**Review by krishi0408 — APPROVED** [review]

**Pull request merged**